### PR TITLE
V1.2 Redesign for Dashboard and Category Details and Tag Details

### PR DESCRIPTION
**Pull request by tiffanyqi**
Dashboard changes:
- line that separates calendar and groups
- changed font
- borders around tables
- removed pie legends
- move generate categories button up to title
- add color of category to category table
- category / tag now scrolls
- change edit/delete buttons to white background buttons
- removed cumulative categories/tags

Category detail changes:
- change font
- cumulative hours / hours per week font smaller
- chart / events have borders
- events are now scrollable
- chart and events are side by side

Tag detail changes:
- keywords, cumulative hours, and hours by week font smaller
- font change
- removed "cumulative" and add quotes to make it consistent with category
- chart / events / pie have borders
- events are now scrollable
- chart events / pie are side by side
- pie is named "Calendars of Tag" and has a border around it
- moved calendar to the bottom (to be removed)

To see it in full, check out my frontend branch!

### CODE REVIEW

**Review by jamesuejio — CHANGES_REQUESTED** [review]
Only request is disable hide hours, so on the main page Tags with 0 hours still show up. This is line 10 in home_logged_in, hide-zero-hours = "false". Everything else looks great. I think after Andrew merges and we have a one page app, we can start nit picking front end changes, but for now it's good. I would also wait until @MaxFangX reviews or I know @andrewhuang has opinions on front end changes.

**Comment by tiffanyqi** [comment]
@jalagar you mean line 80? In any case, done! :)

**Review by cindytung — APPROVED** [review]
omg this looks so much better!!

**Review by jamesuejio — APPROVED** [review]

**Pull request merged**
